_posts/blog/2023-06-20-ghost-buses.md

```diff
@@ -0,0 +1,58 @@
+
+Q. How did the Ghost Buses group fit in with other Chicago transit efforts? 
+
+It's important that new civic tech projects acknowledge and fit into existing civic tech efforts. When the Ghost Buses breakout group started, we did research to identify existing transit efforts like CTAction, the ATA, and CJWJ. We emailed, phone-called, invited representatives to come to CHN to talk about transit in Chicago, and went to events and rallies organized by these organizations. Over time, we've developed a symbiotic relation with all of these transit efforts - we focus mostly on data analysis, but also share resources more broadly. We've learned that it's very important for transit efforts to work together.
+
+Q. "What have we built so far" 
+
+A. At a high level, this project compares real-time data (data that indicates which bus trips actually happened) with schedule data (the trips that the CTA said in advance would happen). 
+
+The real-time data is scraped from the CTA bus tracker API[  getvehicles feed](https://www.transitchicago.com/developers/bustracker/) every five minutes. Data collection started on 20 May 2022 and continues to date, though the data on this site is not yet automatically updated. Schedule data is collected from the [CTA General Transit Feed Specification (GTFS) feed data](https://www.transitchicago.com/developers/gtfs/). To compare the two sources, we look at how many distinct trips occurred each day on each route in the real-time data vs. how many distinct trips were scheduled to occur that day on that route. We can then calculate a reliability percentage showing how many of the scheduled trips were observed in the realtime data.
+
+The data displayed on our website right now is not directly this reliability percentage but instead is the percentile comparing all the routes across the city. Because our original research question was related to the distribution of ghost buses across the city, we have focused on comparisons between routes so far.
+
+Q. "Where is the project going from here"
+
+A. We've got a few ongoing projects! One project we have recently finished is making maps of the reliability of bus service by ward. Wards divide up a city into distinct areas for the purposes of administration and elections. Last May 2022, the wards were redistricted. We are soon releasing the ward map to help citizens better understand how ghost buses affect wards throughout the city. If you're interested in contributing, please visit our [GitHub page](https://github.com/chihacknight/breakout-groups/issues/217), join our [CHN Slack](http://slackme.chihacknight.org/), or come to a [weekly CHN meeting](https://chihacknight.org/)!
```

Comment:
I would actually rework this to say the following (also aligns with Dylan's feedback on Slack):

> We are currently working on route-specific pages on the site, to provide more detail. On the back-end, we are working on further automations of our data pipeline, to make data updates less manual. 


(At this point, I'm not sure anyone is still actively working on the wards feature)